src-docs/src/views/inline_edit/inline_edit_example.js

```diff
@@ -152,6 +163,30 @@ export const InlineEditExample = {
       snippet: inlineEditTitleSnippet,
       playground: inlineEditTitleConfig,
     },
+    {
+      title: 'Controlling inline edit text and title values',
```

Comment:
copy suggestion for brevity
```suggestion
      title: 'Controlled values',
```

Reply:
also, nice job with the organization of this new section! within the flow of the whole page/set of examples, i think it's a great location for it.